grails-app/views/project/index.gsp

```diff
@@ -265,19 +267,20 @@
                         </cl:ifAdmin>
                     </cl:isLoggedIn>
                     <cl:ifValidator project="${projectInstance}">
-                    <g:link class="btn btn-default btn-hollow grey" controller="task" action="projectAdmin"
-                            id="${projectInstance.id}">Validate tasks</g:link>
-                </cl:ifValidator>
-
+                        <g:link class="btn btn-default btn-hollow grey" controller="task" action="projectAdmin"
+                                id="${projectInstance.id}">Validate tasks</g:link>
+                    </cl:ifValidator>
                 </div>
             </div>
         </div>
 
-        %{--<div class="row">--}%
-            %{--<div class="col-sm-12 image-origin float-right">--}%
-                %{--<p>Image by [Name] from Flickr</p>--}%
-            %{--</div>--}%
-        %{--</div>--}%
+        <g:if test="${projectInstance.backgroundImage && projectInstance.backgroundImageAttribution}">
+            <div class="row">
+                <div class="col-sm-12 image-origin">
```

Comment:
The `if` test above will only show this row if there is a `backgroundImage` and a `backgroundImageAttribution`.  The `image-origin` class adds a large top margin, so extends the element the hero image covers.  IMO it would look nicer to include this row if there is a `backgroundImage` even if there is no `backgroundImageAttribution`